package.json

```diff
@@ -5,7 +5,8 @@
   "main": "index.js",
   "scripts": {
     "test": "test",
-    "start": "nodemon src/server/index.js"
+    "start": "nodemon src/server/index.js",
+    "build": "webpack -w"
```

Comment:
Should we go ahead and have a 'production build' script as well?  Do we need to set NODE_ENV or otherwise indicate to babel when to use the 'production' settings specified in .babelrc?